pkg/kubelet/dockershim/helpers_unsupported.go

```diff
@@ -18,6 +18,40 @@ limitations under the License.
 
 package dockershim
 
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+	dockertypes "github.com/docker/engine-api/types"
+	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
+)
+
 func DefaultMemorySwap() int64 {
 	return -1
 }
+
+func (ds *dockerService) getSecurityOpts(containerName string, sandboxConfig *runtimeapi.PodSandboxConfig, separator rune) ([]string, error) {
```

Comment:
this may cause unit tests failure on OSX, could you also move related unit tests to platform specific files?